src/config/dynamic-theme-fixes.config

```diff
@@ -3902,6 +3902,38 @@ INVERT
 
 ================================
 
+jumia.com
+jumia.co
+jumia.ma
+jumia.ci
+jumia.dz
+jumia.ug
+jumia.sn
+zando.co.za
+
+INVERT
+svg.ic[role="img"]:not([aria-label="Zando logo"])
+.logo > a > img
+.inbox > a:not(:nth-of-type(1)) > svg
```

Comment:
Icons in login box (Except the first icon)
![Log in icons](https://user-images.githubusercontent.com/14346945/100476559-8cf2bc80-30ee-11eb-9c3d-a835697bd104.jpg)
![Log in icons (2)](https://user-images.githubusercontent.com/14346945/100476560-8e23e980-30ee-11eb-9b39-d197e0d0a199.jpg)